src/LandingPage.js

```diff
@@ -8,12 +10,24 @@ class LandingPage extends Component {
         this.state={
 
         }
+        var inputString = '';
     }
 
     render(){
         return(
             <div className="landing-page">
                 <img src="codeCollabBanner.svg" className="banner" alt="CodeCollab Logo"/>
+                <div className="section-info centered">
+                <Link to='/collab'>
+                  <div className="fake-button-green centered" onClick={this.getName}>Start a Collab</div>
+                </Link>
+                <p className='bar'>|</p>
+                <input className='section-info round-input' ref={(input) => {this.inputString=input}} placeholder='Your Collab ID...'/>
```

Comment:
Update this to pull from state instead of a local variable

Reply:
fixed this at the same time that I changed the value to be stored in the state object.